scripts/stop.sh

```diff
@@ -0,0 +1,3 @@
+sudo /sbin/stop cor-site
+cd /opt/codingteam/codingteam.org.ru/
+rm -rf *
```

Comment:
I don't know how this script is executed, so I'm assuming it's simply `/bin/sh scripts/stop.sh`. Correct me if I'm wrong.

By default, shells carry on with the operations regardless of errors. If `cd` above fails, shell will print out an error and *proceed to running `rm -rf *` in the current directory*.

There are two ways around that:
1. `set -e` at the beginning of the script. (See [POSIX Shell spec](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#set) for details on that one.)
2. Combine `cd` and `rm` with `&&`: `cd /opt/codingteam/codingteam.org.ru/ && rm -rf *`.

And if you ask me, I'd vote for using *both* approaches here.